src/extensions/cite/index.test.ts

```diff
@@ -0,0 +1,47 @@
+import { whenReady, select, tag, attr } from '../../util'
+
+test('DOM manipulations', async () => {
+  const body = document.body
+  body.innerHTML = `
```

Comment:
Worth considering defining these fixtures externally and importing them? Would make for cleaner test files, like:

```JavaScript
import fixture from './index.fixture.ts'
```
and then in the test:
```JavaScript
body.innerHTML = fixture
```

Upside also is a potential downside in that the HTML fixture is then slightly more removed from the test. It's pick your readability, I guess 🤷‍♂ 😄

Reply:
Good point. However, given that these extensions are intended to be temporary, I'm wary of over-engineering them (including their tests) and would like to be able to remove them easily. Also for these tests, I feel that having the fixture located in the test file is an advantage. So I suggest we keep them the way they are for now.